2024-03-21 20:22:31 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7870983460>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/store/
2024-03-21 20:22:31 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7870983400>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/store/
2024-03-21 20:22:31 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7870982c50>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/store/
2024-03-21 20:22:31 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7870983f40>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-21 20:22:31 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7870983a00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-21 20:22:31 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7870982e30>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-21 20:34:35 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7fe075eaa0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/store/
2024-03-21 20:34:35 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7fe075f790>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/store/
2024-03-21 20:34:35 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7fe075f7f0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/store/
2024-03-21 20:34:35 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7fe075e8f0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/store/
2024-03-21 20:34:35 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7fd087eb00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/store/
2024-03-21 20:34:35 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7ff0efce80>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/store/
2024-03-21 20:34:35 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7fe075efe0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-21 20:34:35 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7fe075f250>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-21 20:34:35 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7fe075f790>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-21 20:34:35 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7fe075e740>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-21 20:34:35 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7fe075f730>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-21 20:34:35 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7fe0779210>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-22 11:13:36 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f78709c50c0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-22 11:13:36 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f78709c4e20>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-22 11:13:36 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f78709c4b50>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-22 11:13:36 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f78706275b0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-22 11:13:36 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7870627f40>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-03-22 11:13:36 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f7870627d00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
